app/src/main/java/com/kickstarter/libs/utils/extensions/StringExt.kt

```diff
@@ -0,0 +1,56 @@
+package com.kickstarter.libs.utils.extensions
```

Comment:
You need to add on top of the package the annotation for being able to use this extension from the Java source code 
`@file:JvmName("StringExt")`
take a look into UriExt.kt or ConfigExtension.kt

- Also a heads up, adding that annotation will force you to not have optional methods in an extension class as the ones you have here String?.isEmpty or String?.isPresent

Reply:
So question about this - the previous functionality is testing if a string is null. I don't want to change the functionality for fear of causing a crash so I'd like to leave the null checks there. Will removing these optionals cause any issues?

Reply:
the you'll have to check only on the java code before using that extension function if its null, and in the kotlin code is it's an optional string use the operator '?'